pkg/core/blockchain.go

```diff
@@ -721,7 +686,7 @@ func (bc *Blockchain) resetStateInternal(height uint32, stage stateChangeStage)
 			blocksCnt, batchCnt, keysCnt int
 		)
 		for i := height + 1; i <= currHeight; i++ {
-			err := cache.DeleteBlock(bc.GetHeaderHash(int(i)))
+			err := cache.DeleteBlock(bc.GetHeaderHash(i))
```

Comment:
Nitpicking, this one belongs to a previous commit.

Reply:
Fixed.